output_template/python/run.py

```diff
@@ -93,12 +96,12 @@ def _init(model, config):
     return nn
 
 
-def _run(nn, image_data):
+def _run(image_data, nn):
```

Comment:
Sorry, I missed this line.
Why did change order of arguments?

Reply:
I changed this line because the similar functions (_pre_process, _run, _post_process) should have same order of argument.
But when I checked my code again, my current code is also not so ruled that I cant recover original code.

Reply:
@iizukak 
The changing order has been recovered to original code